policy/policy_handler/legacy/builder.go

```diff
@@ -0,0 +1,65 @@
+package legacy
+
+import (
+	"github.com/atomist-skills/go-skill/policy/types"
+	"olympos.io/encoding/edn"
+)
+
+func BuildLocalEvalMocks(sb *types.SBOM) map[edn.Keyword]edn.RawMessage {
```

Comment:
Yes, we would like to use the existing `SBOM` struct as a standardised API for synchronous policy evaluation.

That struct encapsulates basically all the useful information that we have about a given image, so we can make changes to policies and introduce new policies without needing to rely on any CLI changes to pass new data.

It also gives us a way of getting local attestations (with the help of containerd) into policy evaluation.

Reply:
Okay, I see the idea now. And now that I think about it, I agree it's the right move as well.

Two follow-up thoughts then:
* This won't be a "legacy" thing if it's how we're planning to do local eval in the long-run, so let's not put it in the legacy package.
* Let's try to remove the corresponding mocking logic from go-skill and `scout-cli-plugin` as we are given opportunities to, so that we don't end up holding onto them for some cross-project version combinations. 
    * The packages query mock in this repo will need to remain to support old CLI versions.
    * The user query mock in this repo is released and used by `non-root-user-skill`, so we'll need to hold onto it for a while as well.
    * The packages mock in `scout-cli-plugin` that I linked above is not used by any released skills, to my knowledge. Once this PR is merged, I'll make sure `policy-vulnerabilities` is updated to use the SBOM version.